src/flow/mh_dofhandler.cc

```diff
@@ -37,7 +46,14 @@ edge_4_loc(nullptr),
 row_4_edge(nullptr),
 edge_ds(nullptr),
 el_ds(nullptr),
-side_ds(nullptr)
+side_ds(nullptr),
+row_4_sing(nullptr),
```

Comment:
We are going to remove mh_dofhandler so no code here please. Probably there should be an xfem_dofhandler. Possibly DiscreteSpace interface is simple enough to support it.